src/Administration/Login/UserService/UserService.php

```diff
@@ -78,7 +78,7 @@ private function searchBySub(ParsedIdToken $parsedToken, string $refreshToken):
     private function searchByEmail(ParsedIdToken $parsedToken, string $refreshToken): ?ExternalAuthUser
     {
         $userData = $this->connection->createQueryBuilder()
-            ->select(['user.id', 'user.email'])
+            ->select(...['user.id', 'user.email'])
```

Comment:
Couldn't we just remove the array braces? Makes it easier to read in my opinion
```suggestion
            ->select('user.id', 'user.email')
```